wemod_launcher/utils/logger.py

```diff
@@ -0,0 +1,80 @@
+import os
+from typing import Optional
+from xdg.BaseDirectory import save_data_path
+from pathlib import Path
+import logging
+from os import environ, getenv
+
+
```

Comment:
I mean its somewhat nice to have all these log-levels, but since there aren't that many log messages in wemod, its not that important, if we have something like this i think by default, everything would be logged, of course you could reduce the log output but if a new user is having trouble is easyest to say send over the log.
Maybe you still had that in mind and are separating these messages so it looks nicer in the log and can be reduced by hand, in that case go on.